### PR TITLE
Feature/geo data api

### PR DESCRIPTION
**Pull request by SnirMordhy**

### CODE REVIEW

File: apiObjects/feed.js [hunk 1]
```diff
@@ -131,8 +131,8 @@ api.add = (data) => {
                 id: tweet.id_str,
                 user_id: tweet.user.id_str,
                 text: tweet.text,
-                place: (tweet.place ? tweet.place.country : null),
-                geo: (tweet.place ? tweet.place.bounding_box.coordinates : null),
+                place: ((tweet.place && (tweet.place.place_type === "country")) ? tweet.place.country : null),
```
Comment (itaihay) [inline apiObjects/feed.js]: No need for that. It will always have a country listed

File: apiObjects/feed.js [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 // Dependencies
 const mongoose = require('mongoose'),
-    Model = mongoose.models.Feed,
+    Model = require('../models/feed'),
```
Comment (itaihay) [inline apiObjects/feed.js]: Why was this changed?

File: apiObjects/feed.js (hunk 1 above)
Comment (SnirMordhy) [reply apiObjects/feed.js]: It seems that some of the data is city typed... so on the map, it marked a small range in the country

File: apiObjects/feed.js (hunk 2 above)
Comment (SnirMordhy) [reply apiObjects/feed.js]: made me some problem in using it in different API classes

Comment (itaihay) [reply apiObjects/feed.js]: I’ll test it on the server. If I see an issue i’ll revert to the old code

File: apiObjects/feed.js (hunk 1 above)
Comment (itaihay) [reply apiObjects/feed.js]: Yea but the country will always be saved. Are you saying the geo data would be of the city?

Comment (SnirMordhy) [reply apiObjects/feed.js]: exactly
